caddytls/config.go

```diff
@@ -418,27 +418,6 @@ var supportedCiphersMap = map[string]uint16{
 	"RSA-3DES-EDE-CBC-SHA":          tls.TLS_RSA_WITH_3DES_EDE_CBC_SHA,
 }
 
-// List of supported cipher suites in descending order of preference.
-// Ordering is very important! Getting the wrong order will break
-// mainstream clients, especially with HTTP/2.
-//
-// Note that TLS_FALLBACK_SCSV is not in this list since it is always
-// added manually.
-var supportedCiphers = []uint16{
```

Comment:
Wow, I guess I must've stopped using this with the 0.9 rewrite and not noticed it.

Makes sense though, I don't see why we would need this now. The order is still important, but on the defaultCiphers slice (below) -- I'll have to update the godoc comments (aside from this) - thanks!